manifests/cf-manifest/cloud-config/020-cf-vm-types.yml

```diff
@@ -23,6 +23,15 @@ vm_types:
         size: 10240
         type: gp2
 
+  - name: large
+    network: cf
+    env: (( grab meta.default_env ))
+    cloud_properties:
+      instance_type: m3.large
```

Comment:
Any reason not to use `m4.large` here.  They're fractionally cheaper than the `m3.large`, and give you an extra 0.5 gig of RAM.